### PR TITLE
#6682 beaker config is moved to .bak by beakerx-install

### PR DESCRIPTION
**Pull request by lmitusinski**

### CODE REVIEW

**Comment by scottdraves** [comment]
now that i think about this, it would be better to just chmod the file.
otherwise they will lose their configs every release.

**Comment by lmitusinski** [comment]
beakerx-install changed to modify file privileges to private on instalation

File: beakerx/beakerx/install.py [hunk 1]
```diff
@@ -127,6 +128,11 @@ def _install_magics():
     file.write("c.InteractiveShellApp.extensions = ['beakerx.groovy_magic']")
     file.close()
 
+def _set_conf_priviliges():
```
Comment (scottdraves) [inline beakerx/beakerx/install.py]: spelling error, should be "privileges".

File: beakerx/beakerx/install.py [hunk 2]
```diff
@@ -127,6 +128,11 @@ def _install_magics():
...
+    config_path = os.path.join(paths.jupyter_config_dir(), 'beakerx.json')
+    if pathlib.Path(config_path).exists():
+        os.chmod(config_path, 0o600)
```
Comment (scottdraves) [inline beakerx/beakerx/install.py]: does this work on windows?

**Pull request merged**